pkgs/sse/lib/src/util/id.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:math' show Random;
+
+/// Generates a pseudo-random ID string with 32 bits of entropy.
+String generateId() {
+  final chars = List<int>.filled(6, 0);
+  final random = Random();
```

Comment:
Would there be any benefit if we used `Random.secure()` instead? I'm guessing no, since this isn't actually being used as a secret.

Reply:
Depends on why an ID is being generated to begin with.

If the goal is just to avoid ID collisions, then it depends on how many simultaneous clients a server will have. Too many, and a 32-bit PRNG generated ID might be insufficient to (statistically) avoid two with the same ID.
Then more randomness is enough, it doesn't have to be secure.

If the goal is to prevent someone deliberately forging the same ID as another client, then being secure might be more important than being more random. Brute force might still not be viable.